cinema/views.py

```diff
@@ -167,6 +199,25 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
```

Comment:
The `@extend_schema` decorator is used correctly to define the schema for the `list` action, providing detailed API documentation.